mlir/lib/Conversion/MemRefToLLVM/AllocLikeConversion.cpp

```diff
@@ -15,17 +15,19 @@
 using namespace mlir;
 
 namespace {
-LLVM::LLVMFuncOp getNotalignedAllocFn(const LLVMTypeConverter *typeConverter,
-                                      Operation *module, Type indexType) {
+FailureOr<LLVM::LLVMFuncOp>
+getNotalignedAllocFn(const LLVMTypeConverter *typeConverter, Operation *module,
```

Comment:
Nit: If you are already changing this, can you make the functions static instead. See comment further down for explanation why.